run_tests.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Would you mind changing this to a [python unit test](https://docs.python.org/3/library/unittest.html)?

I recommend you to use [plumbum](http://plumbum.readthedocs.io/) to execute commands. Combininb plumbum with unit tests is a pleasure ([see an example here](https://github.com/Tecnativa/docker-postgres-autoconf/blob/master/tests/test.py)).

Reply:
Yeah, as mentioned in the PR I wanted to do this eventually. I agree this script isn't very maintainable.